docs/getting_started.rst

```diff
@@ -0,0 +1,170 @@
+
+Using Conda environments is not mandatory to be able to install and use WaterTAP; however, it is strongly recommended. To use Conda environments, the ``conda`` package manager is required. Refer to the `Conda installation guide <https://idaes-pse.readthedocs.io/en/stable/tutorials/getting_started/index.html#installation>`_ for detailed steps on how to install Conda for your operating system.
+
+.. _install:
+
+Install on Linux, Windows, and Mac (ARM)
+----------------------------------------
+
+For Mac users, this section is for Apple's Mac computers with ARM processors (also referred to as Apple Silicon, M1, or M2). To check if your Mac is using an ARM processor, click the apple icon in the upper left corner of your screen. Then select the About This Mac item to view the type of processor in your computer.
+
+Create a Conda environment (in this example, named ``watertap``) where WaterTAP and its runtime dependencies will be installed:
+
+.. code-block:: shell
+
+   conda create --name watertap --yes python=3.8 pip=21.1
+
+Activate the ``watertap`` environment using the command given below. If the environment was activated successfully, the environment's name will be displayed in the terminal prompt such as ``(watertap) project-directory $``.
+
+.. code-block:: shell
+
+   conda activate watertap
+
+Install WaterTAP in the Conda environment using ``pip``:
+
+.. code-block:: shell
+
+   pip install watertap
+
+(Optional) See the :ref:`running-test-suite` section, if you want to verify that the installation was successful.
+
+After installing WaterTAP, the IDAES Extensions command can be used to automatically install the solvers distributed as part of the IDAES Extensions. Depending on your operating system, additional steps might be needed. For more information, refer to the `IDAES installation guide <https://idaes-pse.readthedocs.io/en/stable/tutorials/getting_started/index.html#installation>`_. From the same environment where WaterTAP was installed, run:
+
+.. code-block:: shell
+
+   idaes get-extensions
+
+.. important:: The ``conda activate`` command described above must be run each time a new terminal/console session is started.
+
+.. note:: Typically, the ``idaes get-extensions`` command only needs to be run once for each system, as it will install the required files into a common, system-wide location.
+
+Install on Mac (Intel)
+----------------------
+
```

Comment:
Should we add a warning here about only partial support for Intel Macs? Idaes solvers not withstanding, some models, e.g., the crystallizer, won't work because the rely on IDAES complied extensions.

Reply:
Sure, but I'm not familiar enough with watertap to know what doesn't work on Intel Macs. Can you add a brief warning to the Mac Intel section or post in a comment what you want to include?

Reply:
I don't entirely know myself what will and won't work. So, let's just go with a general warning, e.g. "Intel Mac support is currently experimental. Some WaterTAP features may not work."

Reply:
Done.